src/pages/Typing/Switcher/index.tsx

```diff
@@ -6,9 +6,10 @@ import { useHotkeys } from 'react-hotkeys-hook'
 export type SwitcherPropsType = {
   state: SwitcherStateType
   dispatch: SwitcherDispatchType
+  changeUserPhoneticState: any
```

Comment:
Why `any`?

Reply:
sorry, I'll replace it with correct type later